bus/routes.go

```diff
@@ -585,9 +585,41 @@ func (b *Bus) searchHostsHandlerPOST(jc jape.Context) {
 		return
 	}
 
-	// TODO: on the next major release:
-	// - properly default search params (currently no defaults are set)
-	// - properly validate and return 400 (currently validation is done in autopilot and the store)
+	// validate the usability mode
```

Comment:
Sorry I was a bit too quick asking for a review on this one. Some other stuff come out indeed. I did most of what you suggested but I think we should keep it  `searchHostsHandlerPOST` because we probably did that to avoid query string limits on `KeyIn`?